docs/sources/flow/release-notes.md

```diff
@@ -62,6 +62,11 @@ to `true` and controlled via the `include_scope_labels` argument.
 `name` to `otel_scope_name` and `version` to `otel_scope_version`. This is
 now correct with the OTLP Instrumentation Scope specification.
 
+### Breaking change: `prometheus.exporter.unix` now requires a label.
+
+Previously the exporter was a singleton and did not require a label. The exporter now can be used multiple times and
+needs a label, `prometheus.exporter.unix "example"`.
```

Comment:
Aligning with how other breaking changes are documented (with before/after examples):

``````suggestion
Previously the exporter was a singleton and did not require a label. The exporter now can be used multiple times and
needs a label.

Old configuration example:

```river 
prometheus.exporter.unix { /* ... */ } 
```

New configuration example:

```river 
prometheus.exporter.unix "example" { /* ... */ } 
```
``````